### PR TITLE
add Formats and AdaptiveFormats to Video

### PR DESCRIPTION
**Pull request by Julian-Chu**
# Description

add Formats and AdaptiveFormats to Video and 
expose Formats and AdaptiveFormats of Video to user

## Issues to fix

Please link issues this PR will fix: 
#86

### CODE REVIEW

**Comment by greentornado** [comment]
Please merge bros :D

File: video.go [hunk 1]
```diff
@@ -11,11 +11,13 @@ import (
...
 type Video struct {
-	ID       string
-	Streams  []Stream
-	Title    string
-	Author   string
-	Duration time.Duration
+	ID              string
+	Streams         []Stream
+	Formats         []Format
+	AdaptiveFormats []AdaptiveFormat
```
Comment (corny) [inline video.go:17]: Streams, Formats and AdaptiveFormats seems redundant for me. I'll try to refactor this.

**Comment by corny** [comment]
@Julian-Chu I added another commit. What do you think?

**Comment by Julian-Chu** [comment]
LGTM.

**Comment by greentornado** [comment]
Awesome. Please release then 👍

**Comment by Julian-Chu** [comment]
@greentornado  you can download the release v2 , later I will try to use goreleaser to release same thing again

**Comment by corny** [comment]
I built the binaries with Go 1.15 and uploaded them.

**Comment by corny** [comment]
Strange ... the release data disappared. I did another v2.0.0 release.

**Comment by greentornado** [comment]
I ran `go get -u github.com/kkdai/youtube` but it stucked at 1.2.4 :D

**Pull request merged**